Dockerfile

```diff
@@ -1,12 +1,20 @@
-FROM alpine:edge
+FROM node:8.2-alpine
 
-WORKDIR /code
-ADD . /app
+WORKDIR /usr/src/app
+
+COPY package.json ./
 
-RUN apk --update add nodejs curl && \
-    npm install -g npm && \
-    cd /app && \
+RUN apk --update add curl && \
```

Comment:
Looks like we can get rid of the curl dependency entirely in https://github.com/codeclimate/nsp/pull/1